nextjs_frontend/src/app/agreements/[cpfDevedor]/components/HistoryLineItem/history-line-item.tsx

```diff
@@ -9,47 +9,66 @@ interface HistoryLineItemProps {
 }
 
 export default function HistoryLineItem({
-  divida, proposal, align, connect
+  divida,
+  proposal,
+  align,
+  connect,
 }: HistoryLineItemProps) {
-  const value: number = divida && proposal && proposal.entrada > 0 ?
-    proposal.autor === "User" ? proposal.entrada :
-    divida * proposal.entrada : 0;
+  const value: number =
+    divida && proposal && proposal.entrada > 0
+      ? proposal.autor === "User"
+        ? proposal.entrada
+        : divida * proposal.entrada
+      : 0;
 
   return (
     <div className="w-full flex justify-center gap-5">
       {align === "left" && (
         <>
           <span className="w-1/4 invisible"></span>
-          <HistoryLineConnection accepted={proposal ? proposal.aceito : null} connect={connect} />
+          <HistoryLineConnection
+            accepted={proposal ? proposal.aceito : null}
+            connect={connect}
+          />
         </>
       )}
-      <div className={"w-1/4 flex flex-col " + (align === "left" ? "items-start" : "items-end")}>
+      <div
+        className={
+          "w-1/4 flex flex-col " +
+          (align === "left" ? "items-start" : "items-end")
+        }
+      >
         <h1 className="text-xl font-semibold mb-3">
           {!proposal || proposal.autor === "Bot" ? "Sistema" : "Devedor"}
         </h1>
         {proposal && (
           <>
-            {proposal.entrada > 0 && <span className="text-sm text-zinc-500">
-              <span className="font-medium">Valor de entrada:</span>&nbsp;
-              {value.toLocaleString("pt-BR", {
-                style: "currency",
-                currency: "BRL"
-              })}
-            </span>}
+            {proposal.entrada > 0 && (
+              <span className="text-sm text-zinc-500">
+                <span className="font-medium">Valor de entrada:</span>&nbsp;
```

Comment:
history line description is misaligned (couldn't fix in mobile version -> maybe remove draggable?)